src/styles/sidebar/components/AnnotationHeader.scss

```diff
@@ -23,7 +23,8 @@
 
   // Timestamps are right aligned in a flex row
   &__timestamps {
-    margin-left: auto;
+    flex-grow: 1;
+    text-align: right;
```

Comment:
Because this no longer uses a `margin` rule for alignment, this works fine in combo with the util style.